packages/chaire-lib-common/src/services/routing/OSRMRoutingService.ts

```diff
@@ -88,6 +88,7 @@ export default class OSRMRoutingService extends RoutingService.default {
             const origin = waypoints[0];
             const destination = waypoints[waypoints.length - 1];
 
+            // TODO: TODO: Migrate this value from preferences to config like the osrm modes
```

Comment:
Maybe, but maybe, we don't need the double TODO here :)

Reply:
ok, but this one I'm not 100% sure what's this one is used for, if it's a runtime thing or not. But we can see later